### DISCUSSION TITLE
GradCAM/CAM for multi-image input

### DISCUSSION THREAD

**Discussion by ltsc256** [Project-MONAI/tutorials#1484] [Q&A]
I am using Monai for a classification task that uses both MR and CT data. The classification models work very well and I wanted to implement the interpretability methods (either occlusion sensitivity or cam). However, when I try to run cam, I get an error that forward is missing 1 required argument (the CT image) since the interpretability models seem to only allow 1 image input. Do you have any advice how to handle?

**Comment by KumoLiu** [comment]
Hi, @ltsc256, thanks for your interest here.
I would suggest you separate the inputs in the model. Something like:
```
def model(inputs):
      mri, ct = inputs["mri"], inputs["ct"]
      pass
```
Then you could directly send the inputs to cam/gradcam.

Hope it helps, thanks!
